### ISSUE TITLE
Accessibility Audit: Patient experience flow

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5769]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the patient experience user flow. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/register/<pxp code>`
    * pxp code will depend on org the user is registering for
* `/app/pxp/birth-date-confirmation?plid=<plid>`
    * plid will depend on the result the user is accessing
* `/app/pxp/test-result?plid=<plid>`
     * plid will depend on the result the user is accessing

Use cases to test:
* user registers for a test (in English and Spanish) - inc terms of service
* user attempts to access their results (in English and Spanish) - inc verifying birth date

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by johanna-skylight** [comment]
The ToS page was also checked as part of this ticket!

Below tickets were created to track issues:

https://github.com/CDCgov/prime-simplereport/issues/5831
https://github.com/CDCgov/prime-simplereport/issues/5833
